### ISSUE TITLE
Accidental press Alt+Ctrl+Space triggers unattended installation of unwanted packages with no user confirmation and with no cancel option

### ISSUE THREAD

**Issue by qrp73** [raspberrypi/bookworm-feedback#316]
Yesterday, I encountered an annoying issue: accidental pressing Alt+Ctrl+Space triggers the execution of the keybind command ```gui-pkinst orca reboot```. As a result, this command installs a large number of unwanted packages without any confirmation or option to cancel.

I spent a significant amount of time removing these packages, which also led to the removal of essential packages, requiring additional time to restore the system. All of this happened simply because of an accidental press of Alt+Ctrl+Space.

Could you please fix this issue? 
Ideally, add a prompt requiring a password to confirm package installations initiated through a keybind shortcut. Or just remove these keybind.

As a workaround you can fix it by remove this section from ```/etc/xdg/labwc/rc.xml``` and from ```/etc/xdg/openbox/lxde-pi-rc.xml``` and from ~/.config/labwc/rc.xml:
```
    <keybind key="C-A-space">
      <action name="Execute">
         <command>gui-pkinst orca reboot</command>
      </action>
    </keybind>
```

And remove this section from ```/etc/wayfire/template.ini``` and from ```~/.config/wayfire.ini```:
```
binding_orca = <ctrl> <alt> KEY_SPACE
command_orca = gui-pkinst orca reboot
```

**Comment by lurch** [comment]
If you boot up your Pi for the first time with speakers attached, you'll hear it tell you "Press Ctrl Alt Space to install a screen reader", so I assume that this is intended behaviour (for the benefit of blind and partially-sighted people).

**Comment by qrp73** [comment]
But it should give user the chance to cancel that destructive operation if he pressed it accidentally as in my case. Currently it works as a shortcut for permanent damage to the system with no confirmation and no way to cancel it. It is especially unpleasant to see such system damage if you did not know about this feature and did not disable these shortcuts immediately after installation and catch it during doing some work...

In addition, I must note that the voice reproduces an very annoying text from which it is not clear what is happening. it confuses and hinders more than it helps

**Comment by lurch** [comment]
If you "accidentally" press the power button on your Pi 5 twice in a row, it automatically shuts down without giving you the chance to "cancel"...

You're the first person to ever complain about accidentally pressing Alt+Ctrl+Space so I'm 99.9% sure we won't be changing this behaviour.

**Comment by qrp73** [comment]
I spent about six hours restoring the system just because of accidentally pressing this key combination, which I didn’t even know existed. That’s the issue.

I must say, it looks quite alarming when, while working, a window unexpectedly appears with installation and no option to cancel or understand what’s happening.

My first thought was that it might be a virus or trojan, as it behaves exactly like that and with similar results.

> You're the first person to ever complain about accidentally pressing Alt+Ctrl+Space so I'm 99.9% sure we won't be changing this behaviour.

I hadn’t encountered this issue for over a year simply because I didn’t press this shortcut and didn’t know about it. But now it happened - I pressed it accidentally, and in my case, the result was catastrophic. I got broken system.

This is undoubtedly a true destructive shortcut or disaster-prone key binding.

At first, the system worked, but it startled me with unexpected and scary speech words at every action. After attempting to remove some installed packages, key system components were also deleted, leaving me with a non-functional system that booted into a command line with numerous errors. I managed to restore it, but it took about six hours. I certainly didn’t expect that kind of result from accidentally pressing three keys.

**Comment by spl237** [comment]
You can just uninstall Orca in Recommended Software. No need to rebuild entire systems at all. 

It is unreasonable to expect partially-sighted users to enter passwords or anything of the like. 

It is also not possible to cancel an apt install once it is in progress, or at least not without potentially leaving the system in a broken state.

This behaviour isn’t going to be changed. We have an obligation to make life as easy as possible for physically-disadvantaged users; it is no more than a very occasional minor annoyance for anyone else.

**Comment by qrp73** [comment]
I’ve already removed the keybind for this key combination, so hopefully, this won’t happen again. But it’s definitely a dangerous shortcut. Adding at least a confirmation window could prevent a lot of issues. Unfortunately, by default, this key combination triggers an unconditional installation of components, without asking the user if they really want it and without any option to cancel the operation.

And I’ll note that killing the system with an accidental key press definitely doesn’t make life easier - instead, it complicates it.

**Comment by lurch** [comment]
I don't know what you did wrong whilst trying to uninstall Orca, but please stop describing this as a "destructive operation" which "kills systems".

**Comment by qrp73** [comment]
Users with physical disabilities often depend on consistent and predictable system behavior. Therefore, an unforeseen change in system behavior, especially without prior warning or user consent, is likely to cause greater harm than benefit to these individuals.

Regarding to the function itself, as I mentioned above, the speech prompts didn’t help, as their content seemed more like fragmented and random word outbursts rather than any kind of assistance.

If it was scared even me, I can only imagine the impact it would have on a physically disadvantaged user if such an event were to occur without user consent.

**Comment by spl237** [comment]
Your feedback is noted. 

But the behaviour will not be changed.

**Comment by qrp73** [comment]
> I don't know what you did wrong whilst trying to uninstall Orca, but please stop describing this as a "destructive operation" which "kills systems".

I refer to this as a "destructive operation" because, after enabling this feature, the chaotic word bursts from the speaker left me feeling completely helpless. I was unable to do anything or even analyze what had happened until I turned off the speakers. These random word bursts made it extremely difficult to proceed, I have no idea why, but it works in that way, some kind of hypnotic effect. 

And it completely interrupted my work for 6 hours due to the accidental press of this key combination. This is why I name it "destructive operation".

In my attempt to remove the issue quickly, I accidentally deleted a package, and along with it, critical system services. While I noticed additional packages being removed during the process, I immediately tried to reinstall the deleted packages. Unfortunately, after a reboot, the system was appears inoperable.

I hope this helps clarify my experience, and improve this function. As for me, I just removed this dangerous keybind, because I often can use different key shortcuts and don't want to catch such issue again. I added removing this keybind as one of the first steps in my post-installation guide.

For me, the unexpected installation of any packages without an explicit user request for update or installation (which I intentionally perform only via the console) is an unacceptable system behavior.

> If you boot up your Pi for the first time with speakers attached, you'll hear it tell you "Press Ctrl Alt Space to install a screen reader", so I assume that this is intended behaviour (for the benefit of blind and partially-sighted people).

I always boot my Pi with speakers attached, include first boot after installation. I done installation and first boot many times. But I never hear it.

Moreover, this spontaneous package installation clearly didn’t happen upon the first boot, but rather after about six months of using the system after installation.

I would also like to add that after this system corruption incident, I needed the system and installed a new instance on a new SD card. Upon the first boot, I didn’t hear anything like that, even though my speakers were connected.
